apps/android/LiveVideoApp/app/src/main/java/com/twilio/livevideo/app/manager/GridManager.kt

```diff
@@ -124,13 +158,32 @@ class GridManager {
             val inflater = LayoutInflater.from(gridLayout.context)
             val participantViewBinding: ParticipantViewItemBinding =
                 createParticipantView(inflater, item, gridLayout, isVerticalModeEnabled, lifecycleOwner, indexView)
+
+            item.identity?.apply {
+                val popup = createSpeakerPopup(this, participantViewBinding.participantMenu)
+                participantViewBinding.participantMenu.setOnClickListener {
+                    popup.show()
+                }
+            }
```

Comment:
Is there a good way to move this from the grid code to the participant view code? For these reasons:

1. The grid code would be more single responsibility. It would be responsible for adding, removing, or moving grid items but not the internal behavior of a grid item.
2. When the speaker layout is implemented this popup will need to be handled there also. If the popup code remains in the grid code, it would need to be duplicated for the speaker layout.

Reply:
I have chosen to implement the `Popup menu` here instead of Participant code, because this is related to UI scope. So, in the participant code, I don't keep any code about UI. 

I understand your reasons, so, In that sense, I would prefer to move that code to the Screen code (Aka Fragment), then it can be shared with the `Speaker Layout` in the future.

Reply:
Ok, I will leave it up to you if you want to refactor now or later when you implement the speaker layout.

Reply:
I was checking the impact of these changes. I would prefer to make those changes once I start working on the `Speaker Layout`  implementation.